nautobot/core/utils/git.py

```diff
@@ -34,8 +35,8 @@
 
 def swap_status_initials(data):
     """Swap Git status initials with its equivalent."""
-    initial, text = data.split("\t")
-    return GitDiffLog(status=GIT_STATUS_MAP.get(initial), text=text)
+    initial, text = data.split("\t", 1)
+    return GitDiffLog(status=GIT_STATUS_MAP.get(initial[0]), text=text)
```

Comment:
GIT_STATUS_MAP uses only single characters (`"R"`) but in some cases `initial` could also include a confidence percentage, e.g. `"R100"`